### PR TITLE
TR-4557: Update Fastly HTTPS setup guide

### PR DESCRIPTION
**Pull request by gilesv**
Ticket: https://messagebird.atlassian.net/browse/TR-4557

### What Changed

- Updated Fastly guide on HTTPS Engagement Tracking, with screenshots of the new UI and more details about correctly forwarding the `Host` header to SparkPost.
- Screen recording of the guide:

https://github.com/SparkPost/support-docs/assets/18203276/2ccd30f9-9792-4278-a977-b7bd7bf9a284
- Link to guide in dev: http://localhost:3000/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost#step-by-step-guide-with-fastly

After setting up my tracking domain with HTTPS provided by Fastly, I noticed that it automatically forwards the `Host` if the `Override host` option is disabled:
```
~❯ curl -v https://click.redlabs.shop/f/woohoo/XhBKq2IoQzooHYGhQFI21w\~\~/AAUBzwA\~/RgRmfbURP0RJaHR0cHM6Ly9kb2NzLm5naW54LmNvbS9uZ2lueC9hZG1pbi1ndWlkZS9sb2FkLWJhbGFuY2VyL2h0dHAtbG9hZC1iYWxhbmNlclcDc3BjQgpkkREwm2TECMaQUiN2aW5pY2l1cy5naWxlcyt0ZXN0QG1lc3NhZ2ViaXJkLmNvbVgEAAAAAA\~\~
*   Trying 199.232.113.91:443...
* Connected to click.redlabs.shop (199.232.113.91) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* (304) (OUT), TLS handshake, Client hello (1):
* (304) (IN), TLS handshake, Server hello (2):
* (304) (IN), TLS handshake, Unknown (8):
* (304) (IN), TLS handshake, Certificate (11):
* (304) (IN), TLS handshake, CERT verify (15):
* (304) (IN), TLS handshake, Finished (20):
* (304) (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / AEAD-CHACHA20-POLY1305-SHA256
* ALPN, server accepted to use h2
* Server certificate:
*  subject: CN=click.redlabs.shop
*  start date: Jun 27 17:28:20 2023 GMT
*  expire date: Sep 25 17:28:19 2023 GMT
*  subjectAltName: host "click.redlabs.shop" matched cert's "click.redlabs.shop"
*  issuer: C=US; O=Let's Encrypt; CN=R3
*  SSL certificate verify ok.
* Using HTTP2, server supports multiplexing
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Using Stream ID: 1 (easy handle 0x12780f200)
> GET /f/woohoo/XhBKq2IoQzooHYGhQFI21w~~/AAUBzwA~/RgRmfbURP0RJaHR0cHM6Ly9kb2NzLm5naW54LmNvbS9uZ2lueC9hZG1pbi1ndWlkZS9sb2FkLWJhbGFuY2VyL2h0dHAtbG9hZC1iYWxhbmNlclcDc3BjQgpkkREwm2TECMaQUiN2aW5pY2l1cy5naWxlcyt0ZXN0QG1lc3NhZ2ViaXJkLmNvbVgEAAAAAA~~ HTTP/2
> Host: click.redlabs.shop
> user-agent: curl/7.79.1
> accept: */*
>
< HTTP/2 302
< content-type: text/plain
< location: https://docs.nginx.com/nginx/admin-guide/load-balancer/http-load-balancer
< server: msys-http
< x-robots-tag: noindex
< accept-ranges: bytes
< date: Tue, 27 Jun 2023 18:54:37 GMT
< via: 1.1 varnish
< age: 66
< x-served-by: cache-cgh11176-CGH
< x-cache: HIT
< x-cache-hits: 1
< x-timer: S1687892077.292871,VS0,VE38
< content-length: 0
<
* Connection #0 to host click.redlabs.shop left intact
```
Given this, I updated the guide to mention the absolute need for the `Override host` option to be off as the `Host` must be forwarded to SparkPost.

### How To Test or Verify

- Run it locally and make sure the updates are there

#### All PRs Checklist

- [x] Give your pull request a meaningful name.
- [x] Use lowercase filenames.
- [x] Apply at least one team label according to which team is the content expert (ie. `team-FE` or `team-SAZ`)
- [ ] Pull request approval from the FE team or content experts (see label applied above) that isn't the content creator.

#### Content Changes Checklist

- [x] Check that your article looks correct in the preview here or in a [Netlify deploy preview](https://app.netlify.com/sites/support-docs/deploys).
- [x] Check the links in your article.
- [x] Check the images in your article (if there are any)
- [x] Check to make sure you are using markdown appropriately as outlined in `examples/article.md` in the root of the project directory and on the momentum doc's [preface article](https://support.sparkpost.com/momentum/4/4-preface)
- [ ] Check to make sure the [Copy and Tone Guidelines are followed](https://docs.google.com/document/d/1dej9J7N9M8lcbJXnT9kxyNB_EFBPHIBLZBbHaxRWqhQ/edit).

### CODE REVIEW

File: content/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md [hunk 1]
```diff
@@ -338,72 +338,72 @@ Once your CNAME is set up with your DNS provider, instead of providing an existi
...
+     ![](media/enabling-https-engagement-tracking-on-sparkpost/fastly-2023-add-host.png)
 
-    Fastly detects that SparkPost supports TLS, and shows the host entry like this. Optionally you can use the "pencil" edit icon to set a meaningful name.
+     Fastly detects that SparkPost supports TLS, and shows the host entry like below. Optionally you can use the "pencil" edit icon to set a meaningful name.
 
-    ![](media/enabling-https-engagement-tracking-on-sparkpost/fastly-origin.png)
+     ![](media/enabling-https-engagement-tracking-on-sparkpost/fastly-2023-host-added.png)
 
-    Fastly default settings pass the `user_agent` and `ip_address` through to SparkPost engagement tracking as expected.
+     Fastly forwards the `user_agent` and `host` HTTP headers by default to SparkPost engagement tracking as expected.
```
Comment (lucas-mitri-messagebird) [inline content/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md:361]: Minor thing, I know it was already like this, but while we're at it, can we change `user_agent` to `User-Agent`?

**Review by lucas-mitri-messagebird — APPROVED** [review]
This looks great!

File: content/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md [hunk 2]
```diff
@@ -339,71 +339,72 @@ Once your CNAME is set up with your DNS provider, instead of providing an existi
 
 ---
 ## Step by Step Guide with Fastly
+_Updated June 2023. Images and descriptions follow the current Fastly web UI._
 
-Sign up for Fastly or log in to an existing account.
+Sign up for [Fastly](https://www.fastly.com/) or log in to an existing account.
 
-1. Select the **Configure** tab on the Dashboard, then "Create Service". Give your service a name, and add your tracking domain under "Domains".
+1. Select the **Deliver** tab on the Dashboard, then click the **Create a delivery service** button. Give your service a name in the _Options_ menu and clicking **Edit service name**.
```
Comment (NupurKulkarni1) [inline content/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md:346]: nitpicky - Give your service a name in the Options menu `by` clicking Edit service name.

**Review by NupurKulkarni1 — COMMENTED** [review]
Verified new screenshots by running locally. LGTM!

File: content/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md (hunk 2 above)
Comment (gilesv) [reply content/docs/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md]: Done

**Pull request merged**